.github/workflows/fast-dispatch-build-and-unit-tests.yaml

```diff
@@ -72,12 +69,9 @@ jobs:
         timeout-minutes: ${{ inputs.timeout }}
         uses: ./.github/actions/docker-run
         with:
-          docker_username: ${{ github.actor }}
+          install_wheel: true
           docker_password: ${{ secrets.GITHUB_TOKEN }}
-          docker_image_arch: ${{ inputs.arch }}
           run_args: |
```

Comment:
Could combine this into just one line so it's pretty: `run_args: ${{ matrix.test-group.cmd }}`